src/source/Response.h

```diff
@@ -108,14 +108,6 @@ STATUS freeCurlResponse(PCurlResponse*);
  */
 STATUS closeCurlHandles(PCurlResponse);
 
-/**
- * Debug prints statistics of the request/response
- *
- * @param - PCurlResponse - IN - Response object
- *
- */
-VOID dumpResponseCurlEasyInfo(PCurlResponse);

```

Comment:
I think we should add this back and comment it out like the contents of the function in _Response.c_

Reply:
Already did it.